src/js/flows/initSpace.ts

```diff
@@ -27,10 +27,14 @@ export const initSpace = (spaceId: string): Thunk => (dispatch, getState) => {
       if (!space.hasIndex()) {
         dispatch(SearchBar.setTarget("events"))
       }
-      dispatch(submitSearch({history: true, investigation: false}))
+      dispatch(submitSearch({history: true, investigation: false})).catch(
+        (e) => {
+          dispatch(Notice.set(ErrorFactory.create(e)))
+          throw e
+        }
+      )
```

Comment:
I wanted to specifically catch and ~release~ handle the error from search differently than the outer catch-all here. In other words, if search threw an error, we set the notice and then throw the error again so that the outer catch will also log out that error (and if there were any other non-search thrown errors, then we would still log those). I just realized two things though:

1) I opted to use a simpler statusCheck flow for retrying instead of using this for our retrying, so this flow is no longer used in situations where we want to keep context for the user.

2) If there is a non-network error thrown, we still want to display that in a notice.

So I'll push something up with a single catch at the end like:
```js
    .catch((error) => {
      console.error(error)
      const e = ErrorFactory.create(error)
      if (e.type === "NetworkError") return

      dispatch(Notice.set(ErrorFactory.create(e)))
    })
```